docs/readme.md

```diff
@@ -17,7 +17,6 @@ Start by adding the following to your Cargo.toml file:
 [dependencies.windows]
 version = "0.38.0"
 features = [
-    "alloc",
```

Comment:
Think you want to leave this one, right? Users following along in the README will fail because 0.38 doesn't represent what's in main.

Reply:
Yes, I really should split the readme for the repo from the readme used by crates.io ...